backend/pom.xml

```diff
@@ -66,6 +66,17 @@
             <artifactId>spring-boot-autoconfigure</artifactId>
             <version>2.3.4.RELEASE</version>
         </dependency>
+        <dependency>
```

Comment:
On line 145 (I can't write a comment on the line directly for some reasons):
```
<directory>${pom.basedir}/src/test/java</directory>
```
shouldn't this be 
```
<directory>${pom.basedir}/src/test/groovy</directory>
```
for tests instead?